src/opt.rs

```diff
@@ -0,0 +1,53 @@
+use clap::Parser;
+use std::fmt::Debug;
+use wasm_opt::{OptimizationError, OptimizationOptions};
+
+#[derive(Parser, Debug)]
+pub struct Cmd {
+    /// WASM file to optimize
+    #[clap(long, parse(from_os_str))]
+    wasm: std::path::PathBuf,
+    /// Path to write the optimized WASM file to (defaults to same location as --wasm)
+    #[clap(long, parse(from_os_str))]
```

Comment:
Maybe a safer default is `stdout`, instead of overwriting the input file?

Reply:
Or some default rename `foo.opt.wasm`? `foo.min.wasm` (to match `foo.min.js` convention)?

Reply:
Agreed, this is a really good point, the CLI should not default to an action that is destructive.

Writing to stdout might break peoples shells because it's binary data and not all shells handle that well.

Writing to a alternative file by default seems like a good default to me. I've named it `.optimized.wasm` to match the terminology of the commands name.